src/error.rs

```diff
@@ -21,6 +21,9 @@ pub enum Error {
     #[error("Unable to find configuration '{0}'")]
     UnknownConfiguration(String),
 
-    #[error("No configuration specified and fzf not installed")]
-    NoConfigurationSpecifiedNoFzf,
+    #[error("Unable to setup SkimOptionsBuilder")]
+    SkimBuildError,
```

Comment:
I wonder under what conditions this can actually happen. Like if a user gets this error, what could they reasonably do to make it work? Or is it purely a programming bug so it may be better to panic instead?